src/usng.ts

```diff
@@ -613,12 +613,14 @@ extend(Converter.prototype, {
   UTMUPStoLL(utmupsInput) {
```

Comment:
I recommend documenting this function to at least describe its expected input, which it seems can be either a UTM or UPS formatted string or an object describing UTM or UPS coordinates.